Olymp.Nodes/Master/ConfigurationServerNode.cs

```diff
@@ -54,8 +57,12 @@ protected override (Command cmd, IMessage unencryptedMessage) Handle(BaseMessage
 
                     try
                     {
+                        //Check Permission level of issuer
```

Comment:
We don't have a code of conduct yet, so we follow the c# guidelines. That's definitely just me being capricious btw.
```suggestion
                        // Check Permission level of issuer
```